src/Cli/dotnet/commands/InstallingWorkloadCommand.cs

```diff
@@ -182,6 +214,13 @@ protected async Task<List<WorkloadDownload>> GetDownloads(IEnumerable<WorkloadId
             return ret;
         }
 
+        protected void PrintDownloadLink(IEnumerable<string> packageUrls)
+        {
+            Reporter.WriteLine("==allPackageLinksJsonOutputStart==");
+            Reporter.WriteLine(JsonSerializer.Serialize(packageUrls, new JsonSerializerOptions() { WriteIndented = true }));
+            Reporter.WriteLine("==allPackageLinksJsonOutputEnd==");
```

Comment:
So, I've removed this kind of start/end tags from the other JSON outputting commands in `main` here: https://github.com/dotnet/sdk/pull/35640
@baronfel Would we want to follow that same paradigm here in 8.0.2xx?

Reply:
Yes, 100%!

Reply:
Yes, 100%!